### PR TITLE
argo-cd: add statusbadge option to chart

### PR DESCRIPTION
**Pull request by AntoineDao**
Checklist:

* [x] I have update the chart version in `Chart.yaml` following Semantic Versioning.
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] I have followed the testing instructions in the [contributing guide](https://github.com/argoproj/argo-helm/blob/master/CONTRIBUTING.md).
* [x] I have signed the CLA.

### CODE REVIEW

File: charts/argo-cd/templates/argocd-cm.yaml [hunk 1]
```diff
@@ -45,4 +45,7 @@ data:
 {{- if .Values.config.configManagementPlugins }}
   configManagementPlugins: |
 {{ toYaml .Values.config.configManagementPlugins | indent 4 }}
+{{ -if .Values.config.statusbadge.enabled }}
```
Comment (seanson) [inline charts/argo-cd/templates/argocd-cm.yaml]: Typo here, should be `{{- if`

Comment (seanson) [inline charts/argo-cd/templates/argocd-cm.yaml]: As this is a boolean flag, could you please add a default to `values.yaml` and remove the `if` conditional to just use the value:
`statusbadge.enabled: "{{ .Values.config.statusbadge.enabled }}"`

File: charts/argo-cd/Chart.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: "1.2.4"
+appVersion: "1.2.5"
```
Comment (seanson) [inline charts/argo-cd/Chart.yaml]: Please leave this at `1.2.4` as it reflects the image tag used in this chart version.

File: charts/argo-cd/Chart.yaml [hunk 3]
```diff
@@ -1,5 +1,5 @@
...
 description: A Helm chart for Argo-CD
 name: argo-cd
 version: 0.7.2
```
Comment (seanson) [inline charts/argo-cd/Chart.yaml]: Needs a patch bump to `0.7.3`

**Comment by seanson** [comment]
Hi @AntoineDao, there's been some chart churn over the last week or so. Please rebase your changes and we'll be glad to get this going again!

**Comment by AntoineDao** [comment]
Yep will do first thing tomorrow. Sorry to leave this hanging, I totally forgot about it over the weekend :smile:

File: charts/argo-cd/Chart.yaml (hunk 3 above)
Comment (AntoineDao) [reply charts/argo-cd/Chart.yaml]: Ran a patch bump to `1.0.7`.

File: charts/argo-cd/Chart.yaml (hunk 2 above)
Comment (AntoineDao) [reply charts/argo-cd/Chart.yaml]: Yep my bad. Left the `appVersion` where it was after rebasing.

File: charts/argo-cd/templates/argocd-cm.yaml (hunk 1 above)
Comment (AntoineDao) [reply charts/argo-cd/templates/argocd-cm.yaml]: Done!

**Comment by AntoineDao** [comment]
Right, bit of a slow one on my side, sorry about that. Nice the `1.0.0` merge has happened on the Argo CD chart though. Looking forward to playing with it :smile:

**Comment by seanson** [comment]
Given that the new argocd-cm lives in [templates/argocd-configs](https://github.com/argoproj/argo-helm/blob/master/charts/argo-cd/templates/argocd-configs/argocd-cm.yaml) I believe this change should only need a new default value in values.yaml and a chart version bump. Could you also please recommit your README.md changes without formatting changes.

**Comment by AntoineDao** [comment]
Ah hadn't noticed that change as part of the 1.0. To be honest at this point this PR is a bit pointless and I don't think a default value is required as the default should be `don't allow statusbadges` which is done but not specifying it in the Helm chart. 

Thanks for holding my hand through this process and sorry about the faff with the back and forth around formatting. I'll be better next time :bowing_man: !

**Comment by seanson** [comment]
No worries, always glad to have folks around who are willing to help improve things!

**Pull request closed without merging**